src/Command/BehatCommand.php

```diff
@@ -65,6 +46,7 @@ protected function configure(): void
             ->addOption('name', null, InputOption::VALUE_REQUIRED, 'Behat name option to use', '')
             ->addOption('start-servers', null, InputOption::VALUE_NONE, 'Start Selenium and PHP servers')
             ->addOption('auto-rerun', null, InputOption::VALUE_REQUIRED, 'Number of times to rerun failures', 2)
+            ->addOption('selenium', null, InputOption::VALUE_NONE, 'Selenium Docker image')
```

Comment:
Oh, sorry for not detecting this earlier. I don't think that `InputOption::VALUE_NONE` is correct here. If the option is used, a value (valid selenium image) is required.

Reply:
I'm not sure why the tests are passing, somehow thy are bogus and not validating the `VALUE_NONE` restriction that should avoid that option to be passed with any value.

Reply:
Thanks for spotting that, it seems like I hadn't completely understood how those constants worked. I fixed it now.

Not sure about how to improve the tests, though. But I guess it's not related to this feature so if you think it's important to look into that I'd suggest opening a separate issue.

Reply:
Yeah, will check why the tester is not catching that `VALUE_NONE` "violation". It should. But it's apart from this PR, 100% agree.